js/arena.js

```diff
@@ -0,0 +1,38 @@
+import Gladiator from  "./gladiator.js";
+
+export default class Arena {
+    constructor(name){
+        this.name= name.toUpperCase()
+        this.gladiators =[];
+    }
+
+    // To add a new Gladiator
+    addGladiator(newGlad){
+this.gladiators.length <2 ? this.gladiators.push(newGlad) : console.log("You have Two gladiators");
+    }
+
+
+    fight(){
+       if(this.gladiators[0].weapon =="Trident" && this.gladiators[1].weapon =="Spear")
```

Comment:
all the fight conditions here are always removing the first element of the gladiator array to be removed always.

one way to remove an element is to use a method called .splice()
here is a link on how to use it
[splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)

or smart use of pop() and shift() will solve this problem